BinaryObjectScanner/Packer/Crunch.cs

```diff
@@ -0,0 +1,26 @@
+using BinaryObjectScanner.Interfaces;
+using SabreTools.Serialization.Wrappers;
+
+namespace BinaryObjectScanner.Packer
+{
+    // Packer used by all known SmartE games, but also used by some other non-SmartE protected software as well.
+    // https://web.archive.org/web/20020806102129/http://www.bit-arts.com/windows_solutions.html
+    // TODO: Other BitArts products may also use this same string. No samples have yet been found.
+    public class Crunch : IExecutableCheck<PortableExecutable>
```

Comment:
Even though it can't be extracted right now, can you please add an implementation of `IExtractableExecutable` instead? You can see `AdvancedInstaller` for an example of how to format this.

Reply:
Fixed.